server/models/handlers/appointment.js

```diff
@@ -48,8 +48,35 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     }
 }
 
+async function cancelSpecificAppointment(appointmentId) {
+    // If this method is somehow called without specifying values for the required parameters, false is returned.
+    if (_.isNil(appointmentId)) return false;
+    try {
+        const queryOutput = await appointmentModel.returnAppointment(appointmentId);
+        //Check if appt exists
+        const studentId = _.map(queryOutput, 'student_id');
+        if (_.isEmpty(studentId)) return false;
+        
+        // Cancels specific appointment.
+        const cancelSpecificAppointment = await appointmentModel.cancelSpecificAppointment(appointmentId);
+
+        //Finds corresponding workerTimeslotId 
+        const currentWorkerTimeslotId = _.map(queryOutput, 'worker_timeslot_id');
+        const newStatus = TimeslotStatus.available;
+        // Updates the worker's availability to unavailable for the  timeslot of that specific appointment
```

Comment:
Hi Peter,
Not sure if this is a typo, but the comment (and I also agree) that the worker availability would be mapped to `unavailable`. However, on line 65, the `newStatus` is currently set to `available` rather than `unavailable`.

Reply:
I believe this ticket is referring to cancelling appointments, therefore this method should set an unavailable timeslot to `available`.

Reply:
I will update the comment to reflect that